src/common/navigation/MainToolbar.tsx

```diff
@@ -8,6 +8,8 @@ import BasicBreadcrumbs from '@common/navigation/Breadcrumbs'
 
 const MainToolbar: React.FC = () => {
   const {data: session} = useSession();
+  console.log('MainToolbar: session: ');
```

Comment:
Best to not merge until you're ready to remove these! :)

Reply:
Agreed. :)